test/extended/util/framework.go

```diff
@@ -274,6 +274,16 @@ func WaitForADeployment(client kclient.ReplicationControllerInterface, name stri
 	}
 }
 
+// WaitForADeploymentToComplete waits for a deployment to complete.
+func WaitForADeploymentToComplete(oc *CLI, name string) bool {
```

Comment:
Hmm, the namespace is not an argument, the replication controller is:

```go
WaitForADeployment(
  oc.KubeREST().ReplicationControllers(oc.Namespace()),
  name,
  exutil.CheckDeploymentCompletedFn,
  exutil.CheckDeploymentFailedFn)

// vs.

WaitForADeploymentToComplete(oc.KubeREST().ReplicationControllers(oc.Namespace()), name)
```

I thought `WaitForADeploymentToComplete` is just a convenient way to call `WaitForADeployment` with predefined values for the last two arguments.

------------------

As a separate issue, looks like the return type here is wrong, since `WaitForADeployment` returns `error` and not `bool`.